lib/gateway/build-gateway.js

```diff
@@ -614,6 +614,9 @@ async function buildGateway (serviceMap, gatewayOpts, app, lruGatewayResolvers)
 
       defineResolvers(schema, typeToServiceMap, serviceMap, typeFieldsToService, factory, lruGatewayResolvers)
 
+      this.schema = schema
```

Comment:
why do you need to set an instance field with the value of the schema?

Reply:
Correct 👍